### PR TITLE
Bring back volume up/down button functionality on iPhone

### PR DESCRIPTION
**Pull request by wutschel**
## Description

Takes care of feedback from [this forum post](https://forum.kodi.tv/showthread.php?tid=359717&pid=3036679#pid3036679). This PR brings back the volume up/down button functionality for iPhone.

Details:
- Re-enable volume plus/minus button for iPhone
- Ensure correct position and aspect ratios of title (iPad) and image (iPhone)

## Summary for release notes

Improvement: Bring back volume up/down button functionality on iPhone

### CODE REVIEW

File: XBMC Remote/VolumeSliderView.m [hunk 1]
```diff
@@ -94,11 +91,19 @@ - (id)initWithFrame:(CGRect)frame{
...
             muteButton.frame = frame_tmp;
             
-            frame_tmp = volumeSlider.frame;
+            frame_tmp = minusButton.frame;
             frame_tmp.origin.x = muteButton.frame.origin.x + muteButton.frame.size.width + VOLUMEICON_PADDING;
-            frame_tmp.size.width = self.frame.size.width - frame_tmp.origin.x - ANCHORRIGHTPEEK - VOLUMEICON_PADDING;
+            minusButton.frame = frame_tmp;
+            
+            frame_tmp = volumeSlider.frame;
+            frame_tmp.origin.x = minusButton.frame.origin.x + minusButton.frame.size.width;
```
Comment (kambala-decapitator) [inline XBMC Remote/VolumeSliderView.m:99]:
```suggestion
            frame_tmp.origin.x = CGRectGetMaxX(minusButton.frame);
```
please also adjust in other places of this PR

File: XBMC Remote/VolumeSliderView.m [hunk 2]
```diff
@@ -107,15 +112,24 @@ - (id)initWithFrame:(CGRect)frame{
...
             
             img = [UIImage imageNamed:@"volume_1"];
             img = [utils colorizeImage:img withColor:[UIColor grayColor]];
-            [volumeSlider setMinimumValueImage:img];
+            [minusButton setImage:img forState:UIControlStateNormal];
+            [minusButton setImage:img forState:UIControlStateHighlighted];
+            [minusButton setBackgroundImage:nil forState:UIControlStateNormal];
+            [minusButton setBackgroundImage:nil forState:UIControlStateHighlighted];
+            [minusButton setTitle:@"" forState:UIControlStateNormal];
+            [minusButton setTitle:@"" forState:UIControlStateHighlighted];
```
Comment (kambala-decapitator) [inline XBMC Remote/VolumeSliderView.m:123]: why is this needed? These are already default values. (also for plusButton)

**Review by kambala-decapitator — CHANGES_REQUESTED** [review]

File: XBMC Remote/VolumeSliderView.m (hunk 2 above)
Comment (wutschel) [reply XBMC Remote/VolumeSliderView.m:123]: This is required because background images and titles are defined in the xib. For iPhone we need to delete them.

Comment (kambala-decapitator) [reply XBMC Remote/VolumeSliderView.m:123]: clear, thanks. Then better use `nil` for title as well.

File: XBMC Remote/VolumeSliderView.m (hunk 1 above)
Comment (wutschel) [reply XBMC Remote/VolumeSliderView.m]: Agreed, much better readable.

File: XBMC Remote/VolumeSliderView.m (hunk 2 above)
Comment (wutschel) [reply XBMC Remote/VolumeSliderView.m]: Done
